### PR TITLE
Individual enrollments updates

### PR DESCRIPTION
**Pull request by viv-liu**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [ ] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [ ] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Review by avneet723 — APPROVED** [review]

File: specification/iotcentral/data-plane/Microsoft.IoTCentral/preview/2019-10-28-preview/iotcentral.json [hunk 1]
```diff
@@ -1327,18 +1674,42 @@
...
         "summary": "Get device properties",
         "description": "Get all property values of a device by device ID.",
-        "parameters": [
-          {
-            "$ref": "#/parameters/DeviceId"
-          }
-        ],
         "x-ms-examples": {
-          "Get properties": {
+          "Update properties": {
```
Comment (lmazuel) [inline specification/iotcentral/data-plane/Microsoft.IoTCentral/preview/2019-10-28-preview/iotcentral.json]: Why? Seems to still be a GET here.

File: specification/iotcentral/data-plane/Microsoft.IoTCentral/preview/2019-10-28-preview/iotcentral.json [hunk 2]
```diff
@@ -1278,9 +1642,6 @@
               "$ref": "#/definitions/DeviceCommandCollection"
             }
           }
-        },
-        "x-ms-pageable": {
```
Comment (lmazuel) [inline specification/iotcentral/data-plane/Microsoft.IoTCentral/preview/2019-10-28-preview/iotcentral.json]: Why do you remove this? For what I see "DeviceCommandCollection" has nextLink/value as expected for a pageable operation. Seems inconsistent to me.

File: specification/iotcentral/data-plane/Microsoft.IoTCentral/preview/2019-10-28-preview/iotcentral.json [hunk 3]
```diff
@@ -1155,9 +1451,6 @@
               "$ref": "#/definitions/RoleCollection"
             }
           }
-        },
-        "x-ms-pageable": {
-          "nextLinkName": null
```
Comment (lmazuel) [inline specification/iotcentral/data-plane/Microsoft.IoTCentral/preview/2019-10-28-preview/iotcentral.json]: Why do you remove this? For what I see "RoleCollection" has nextLink/value as expected for a pageable operation. Seems inconsistent to me.

File: specification/iotcentral/data-plane/Microsoft.IoTCentral/preview/2019-10-28-preview/iotcentral.json [hunk 4]
```diff
@@ -1007,9 +1213,6 @@
               "$ref": "#/definitions/DeviceTemplateCollection"
             }
           }
-        },
-        "x-ms-pageable": {
-          "nextLinkName": "nextLink"
```
Comment (lmazuel) [inline specification/iotcentral/data-plane/Microsoft.IoTCentral/preview/2019-10-28-preview/iotcentral.json]: See my other comments on paging

File: specification/iotcentral/data-plane/Microsoft.IoTCentral/preview/2019-10-28-preview/iotcentral.json [hunk 5]
```diff
@@ -899,25 +1108,22 @@
               "$ref": "#/definitions/ContinuousDataExportCollection"
             }
           }
-        },
-        "x-ms-pageable": {
-          "nextLinkName": "nextLink"
```
Comment (lmazuel) [inline specification/iotcentral/data-plane/Microsoft.IoTCentral/preview/2019-10-28-preview/iotcentral.json]: See my other comments on paging

File: specification/iotcentral/data-plane/Microsoft.IoTCentral/preview/2019-10-28-preview/iotcentral.json [hunk 6]
```diff
@@ -615,52 +857,121 @@
...
+            }
+          }
+        },
+        {
+          "type": "object",
+          "properties": {
+            "severity": {
+              "description": "Specifies the severity of the event.",
+              "type": "string",
+              "enum": [
```
Comment (lmazuel) [inline specification/iotcentral/data-plane/Microsoft.IoTCentral/preview/2019-10-28-preview/iotcentral.json]: Please use a "x-ms-enum" with a modelAsString=True

File: specification/iotcentral/data-plane/Microsoft.IoTCentral/preview/2019-10-28-preview/iotcentral.json [hunk 7]
```diff
@@ -47,41 +47,194 @@
...
+            }
+          }
+        },
+        {
+          "type": "object",
+          "properties": {
+            "commandType": {
+              "description": "The call pattern of this command.",
+              "type": "string",
+              "enum": [
```
Comment (lmazuel) [inline specification/iotcentral/data-plane/Microsoft.IoTCentral/preview/2019-10-28-preview/iotcentral.json]: Please model enums with x-ms-enum and modelAsString=True

**Review by lmazuel — CHANGES_REQUESTED** [review]
- Paging issues
- x-ms-enum issues

**Review by lmazuel — APPROVED** [review]

**Pull request merged**